de.nicokimmel.shadowcast-electron.json

```diff
@@ -0,0 +1,25 @@
+{
+  "app-id": "de.nicokimmel.shadowcast-electron",
+  "runtime": "org.electron.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.electron.Sdk",
+  "command": "shadowcast-electron",
+  "finish-args": [
+    "--device=all",
+    "--socket=pulseaudio"
```

Comment:
it doesn't display anything?

Reply:
Is there any way to get both x11 and wayland? Since it should support both of them.

Reply:
If you support native wayland use `fallback-x11` and `wayland`, otherwise keep it `x11`